pkg/util/config/config.go

```diff
@@ -33,6 +34,8 @@ type Resources struct {
 	PodEnvironmentConfigMap string            `name:"pod_environment_configmap" default:""`
 	NodeEOLLabel            map[string]string `name:"node_eol_label" default:"lifecycle-status:pending-decommission"`
 	NodeReadinessLabel      map[string]string `name:"node_readiness_label" default:"lifecycle-status:ready"`
+	MaxInstances            int32             `name:"max_instances" default:"-1"`
```

Comment:
shall we comment here on the special meaning of -1 ?

Reply:
That went to the docs.